fe/fe-core/src/main/java/org/apache/doris/backup/BackupHandler.java

```diff
@@ -269,10 +271,30 @@ private void backup(Repository repository, Database db, BackupStmt stmt) throws
                     + " is read only");
         }
 
+        // Determine the tables to be backed up
+        Set<String> tableNames = Sets.newHashSet();
+        AbstractBackupTableRefClause abstractBackupTableRefClause = stmt.getAbstractBackupTableRefClause();
+        if (abstractBackupTableRefClause == null) {
+            tableNames = db.getTableNamesWithLock();
+        } else if (abstractBackupTableRefClause != null && abstractBackupTableRefClause.isExclude()) {
+            tableNames = db.getTableNamesWithLock();
+            for (TableRef tableRef : abstractBackupTableRefClause.getTableRefList()) {
+                tableNames.remove(tableRef.getName().getTbl());
```

Comment:
Do we need to check if there "excluded" tables exist in database?

eg:
```suggestion
                if (!tableNames.remove(tableRef.getName().getTbl())) {
                    throw exception()
                }
```